doc/7-chips/paula.md

```diff
@@ -0,0 +1,20 @@
+# MOS 8364 R4 "Paula"
```

Comment:
amiga.md

Sadly, the Amiga sound system consists of two chips.

Reply:
Easily added!

Reply:
What? I am confused.

Reply:
If there's a second chip in use (which one?) it could easily be added to the docs. If they never work independently, it'd make sense to pair them up into a single "chip".

Mostly I want to avoid situations like having separate YM2612 and Genesis docs as we do now. There's nothing unique to the Genesis that isn't covered by each chip's page individually.

Reply:
The YM2612 and Genesis situation is a remnant of the 0.5 era. Getting rid of Genesis would be the best option.

In the case of Amiga, the problem is that the other chip (Agnus) does not produce sound by itself either. It works in tandem with Paula to output sound.
This is why I request you to keep amiga.md intact.